### PR TITLE
Ana/try to fix mem leak: maybe from network fees

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

I think the slow mem leak we are experiencing could come from the gas estimates / prices coming from DB.

Looking at the code seems to be good, but I believe it might be related because of the times in Digital Ocean graphics and the time this PR, #5056 , was merged

image

image

Not 100% the same but this is the most likely one :shrug: 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Well, the possible failure would be that this query about fees is being done too often and this is causing DB to fail or slow it down, and that is what is messing with API too. Because API is too dependent on DB. So maybe this fix is good, could be.

What do you think?

File: api/data/network-fees.js [hunk 1]
```diff
@@ -60,24 +60,28 @@ const pollForNewFees = async () => {
...
+      networkGasPricesDictionary[networkGasPrice.id].push({ denom: networkGasPrice.denom,
+        gasPrice: networkGasPrice.gasPrice
+      })
     })
-  })
+  }
   // update emoneyGasPrices
   networkGasPricesDictionary['emoney-mainnet'] = emoneyGasPrices
-  return networkGasPricesDictionary[networkId] || []
+  return networkGasPricesDictionary[networkId]
```
Comment (jbibla) [inline api/data/network-fees.js:74]: why this change?

Comment (Bitcoinera) [reply api/data/network-fees.js:74]: because now it cannot be `undefined`

**Comment by jbibla** [comment]
ok let's merge to staging and see if it helps

**Review by jbibla — APPROVED** [review]
